locale/en/LC_MESSAGES/django.po

```diff
@@ -1875,7 +1875,7 @@ msgid "Go to All Invoices page for this Project Space"
 msgstr ""
 
 #: corehq/apps/accounting/templates/accounting/partials/invoice_table.html:8
-#: corehq/apps/hqadmin/templates/hqadmin/hqadmin_base_filters.html:9
+#: corehq/apps/hqadmin/templates/hqadmin/bootstrap2/hqadmin_base_filters.html:9
```

Comment:
Curious why these changes are necessary, I thought the .po files are automatically generated...?